src/app/public/plugin-resources/code-examples/timepicker/basic/timepicker-demo.component.html

```diff
@@ -0,0 +1,42 @@
+<form
+  novalidate
+  [formGroup]="formGroup"
+>
+  <sky-timepicker
+    #timepicker
+  >
+    <input
+      formControlName="time"
+      type="text"
+      [skyTimepickerInput]="timepicker"
+    />
+  </sky-timepicker>
+
+  <p *ngIf="timeControl.errors && (timeControl.dirty || timeControl.touched)"
+    class="sky-error-label"
+  >
+    <span [hidden]="timeControl.errors && !timeControl.invalid">
+      Enter a valid time.
+    </span>
+
+    <span *ngIf="timeControl.errors.required">
+      Time is required. Select a value.
+    </span>
+
+    <span *ngIf="timeControl.errors.skyTime">
+      The time is invalid. Enter a valid time or use the picker.
+    </span>
+  </p>
+</form>
+
+<p>
+  Selected time is {{ timeControl.value ? timeControl.value.local : '' }}
```

Comment:
What is the point of this in the code sample? I think this could make sense in the demo, but is this something we really want to include in our code sample showing how to implement the component?

Reply:
Same question about the clear selection button.

Reply:
Nevermind. Looking in Stackblitz now and realizing this is for the Stackblitz demo. I'll just resolve this comment thread as a non-issue.

Reply:
Removed.